Wikipedia/Code/Session.swift

```diff
@@ -238,8 +238,7 @@ import Foundation
                 }
             }
             
-            if let _ = error,
-            request.cachePolicy != .reloadIgnoringLocalCacheData {
```

Comment:
This fixes the regression issue mentioned in https://github.com/wikimedia/wikipedia-ios/pull/3479.